server/views/components/summary-card.njk

```diff
@@ -6,9 +6,9 @@
                 <h2 class="summary-card__title">
                     {{ params.titleHtml | safe if params.titleHtml else params.titleText }}
                 </h2>
-        {% if params.actions.items.length %}
+        {% if params.action.items %}
```

Comment:
Will only ever be one action link so null check rather than length check of array makes more sense.